lib/beacon/config.ex

```diff
@@ -63,7 +63,10 @@ defmodule Beacon.Config do
   """
   @type media_type_config :: [
           {:backends, list(backend :: module())},
-          {:validations, list(validation_fun :: (Ecto.Changeset.t(), Beacon.Admin.MediaLibrary.FileMetadata.t() -> Ecto.Changeset.t()))}
+          {:validations, list(validation_fun :: (Ecto.Changeset.t(), Beacon.Admin.MediaLibrary.UploadMetadata.t() -> Ecto.Changeset.t()))},
+          {:processor,
+           {prossesor_fun ::
```

Comment:
If I'm not wrong, I see this fn called with 1 arg. Is this spec correct?

Reply:
You are correct. I hadn't updated that to my final implementation. Great catch!